tasks/test/mocha-chrome.js

```diff
@@ -0,0 +1,51 @@
+/* Grunt Task to test mocha in a local Chrome instance */
+
+const MochaCrome = require('mocha-chrome');
+
+module.exports = function(grunt) {
+  grunt.registerMultiTask('mochaChrome', async function() {
+    var done = this.async();
+
+    try {
+      var options = this.data.options;
+
+      for (var testURL of options.urls) {
+        var my_opts = Object.assign({}, options);
+        my_opts.urls = undefined;
+        my_opts.url = testURL;
+        my_opts.chromeFlags = ['--no-sandbox', '--mute-audio'];
+        my_opts.logLevel = 'trace';
+        var success, failure;
+        var runner = new MochaCrome(my_opts);
+        runner.on('ended', stats => {
+          if (stats.failures) {
+            failure = stats;
+          } else {
+            success = stats;
+          }
+        });
+        runner.on('failure', e => (failure = e));
+
+        (async function() {
+          await runner.connect();
+          await runner.run();
+        })();
+
+        await new Promise((resolve, reject) => {
+          runner.on('exit', function() {
+            if (success) resolve(success);
+            else reject(failure);
+          });
+        });
```

Comment:
Maybe split this up to seperate what the functions are here:

```javascript
await new Promise(resolve => runner.on('exit', resolve));

if (failure) {
  throw failure;
}
```

`success` can just be ignored because the reporter will ignore it if we returned it anyway

Reply:
I'm not sure, does this still work around the browser-closing issue?

Reply:
It should run equivalently, yes. But testing it would be good